tests/test_exposition.py

```diff
@@ -79,25 +80,31 @@ def test_escaping(self):
         self.assertEqual(b'# HELP cc A\\ncount\\\\er\n# TYPE cc counter\ncc{a="\\\\x\\n\\""} 1.0\n', generate_latest(self.registry))
 
     def test_nonnumber(self):
-        class MyNumber():
-            def __repr__(self):
-                return "MyNumber(123)"
-            def __float__(self):
-                return 123.0
-        class MyCollector():
-            def collect(self):
-                metric = Metric("nonnumber", "Non number", 'untyped')
-                metric.add_sample("nonnumber", {}, MyNumber())
-                yield metric
         self.registry.register(MyCollector())
         self.assertEqual(b'# HELP nonnumber Non number\n# TYPE nonnumber untyped\nnonnumber 123.0\n', generate_latest(self.registry))
 
 
+class MyNumber(object):
```

Comment:
I'd prefer to keep these where they are so it's clear what's going on.

Reply:
I think inlining helper classes to test methods is generally a bad idea as it pushes test authors to making bigger more complex methods to avoid duplicating components.

That said, have reversed the change and will push now.